extensions/smallrye-graphql-client/runtime/src/main/java/io/quarkus/smallrye/graphql/client/runtime/GraphQLClientConfig.java

```diff
@@ -52,38 +52,69 @@ public class GraphQLClientConfig {
 
     /**
      * The trust store location. Can point to either a classpath resource or a file.
+     *
+     * @deprecated This configuration property is deprecated. Consider using the Quarkus TLS registry.
+     *             Set the desired TLS bucket name using the following configuration property:
+     *             {@code quarkus.smallrye-graphql-client."client-name".tls-bucket-name}.
      */
     @ConfigItem
+    @Deprecated
```

Comment:
Please add `since` and `forRemoval`. Valid for all the configuration properties you deprecated.